warehouse/keyword-common/src/main/java/datawave/util/keyword/TagCloudEntry.java

```diff
@@ -124,6 +127,7 @@ public TagCloudEntry build() {
     public static class ScoreTuple implements Comparable<ScoreTuple> {
         final String source;
         final double score;
+        // TODO-crwill9 rename to partition
         final String language;
```

Comment:
I think we should attempt to remove `language` from `ScoreTuple` instead of renaming it to `partition`. I'm not sure why it's there in the first place because we're already grouping `ScoreTuple`s by language/partition in `TagCloud.addInput(..)`